#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_jhead.1.dpatch by  <rousseau@localhost.localdomain>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP:  Closes: #435072: "jhead: date+time format for -da needs / separator"

@DPATCH@
--- a/jhead.1
+++ b/jhead.1
@@ -214,7 +214,7 @@
 
 Default camera date is 2002:01:01, and date was reset on 2005:05:29 at 11:21 am
 .br
-jhead -da2005:05:29+11:21-2002:01:01
+jhead -da2005:05:29/11:21-2002:01:01
 .TP
 .B \-ts
 Sets the time stored in the Exif header to what is specified on the command line.
